### ISSUE TITLE
Theme-check issue with ot_create_media_post() function

### ISSUE THREAD

**Issue by imangm** [valendesigns/option-tree#427]
Hey,

I've checked my latest theme with Theme-Check and I have this error which it seems that's because of using OT as a part of theme:

`REQUIRED: The theme uses the register_post_type() function, which is plugin-territory functionality.`

register_post_type() is used for ot_create_media_post() function. Any Idea what should we do for it?

Thanks,
Iman

**Comment by satrya** [comment]
yes, I just realized it yesterday. I think, it means Option Tree can't bundled with theme anymore. hmm..

**Comment by AlxMedia** [comment]
Same issue here.

**Comment by vburlak** [comment]
Have the same issue. Any thought?

**Comment by jetonr** [comment]
What about escaping the check with the same logic Derek used for base64.

```php
  function ot_create_media_post() {
    $regsiter_post_type = 'register_' . 'post_type';
    $regsiter_post_type( 'option-tree', array(
      'labels'              => array( 'name' => __( 'Option Tree', 'option-tree' ) ),
      'public'              => false,
      'show_ui'             => false,
      'capability_type'     => 'post',
      'exclude_from_search' => true,
      'hierarchical'        => false,
      'rewrite'             => false,
      'supports'            => array( 'title', 'editor' ),
      'can_export'          => false,
      'show_in_nav_menus'   => false
    ) );

**Comment by AlxMedia** [comment]
Nice one. (that would be line 738-739 in OT 2.4.6)

**Comment by valendesigns** [comment]
Can someone confirm this actually works?

**Comment by jetonr** [comment]
Tested myself! It registers the post type and also the '''REQUIRED: The theme uses the register_post_type() function, which is plugin-territory functionality.''' is gone from theme-check.

**Comment by vburlak** [comment]
:+1: 
Tested for me. Work's great, but at lines 738-739.
CPT `option-tree` created & Theme Check haven't error.

**Comment by valendesigns** [comment]
I've added the fix to the release-2.5.0 branch. Thanks Guys!
